notebooks/rendered/executed/Storage Client Library.md

```diff
@@ -0,0 +1,172 @@
+
+# Google Cloud Storage
+
+This page shows how to get started with the Google Cloud Storage Python client library.
+
+## Import the library
+
+
+```python
+from google.cloud import storage
+```
+
+## Projects
```

Comment:
Could you instead create a section for Key Terms and define: Projects, Buckets, Objects?

Reply:
Reworked this section. Let's take another look tomorrow.